components/Feedback.js

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+import { useRouter } from "next/router";
+
+export default function Feedback(className = '') {
+    const [Submit, setSubmit] = useState(false);
```

Comment:
Please try to not use uppercase for React states. Additionally, it should be an adjective:

```suggestion
    const [submitted, setSubmitted] = useState(false);
```